frontend/packages/db-structure/src/parser/__tests__/testcase.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  type Table,
+  aColumn,
+  aDBStructure,
+  aTable,
+} from '../../schema/index.js'
+
+const userTable = (override?: Partial<Table>) =>
+  aDBStructure({
+    tables: {
+      users: aTable({
+        name: 'users',
+        columns: {
+          id: aColumn({
+            name: 'id',
+            type: 'bigserial',
+            notNull: true,
+            primary: true,
+            unique: true,
+          }),
+          ...override?.columns,
+        },
+        indices: {
+          ...override?.indices,
+        },
+        comment: override?.comment ?? null,
+      }),
+    },
+  })
+
+export const parserTestCases = {
+  'table comment': userTable({
+    comment: 'store our users.',
+  }),
+}
```

Comment:
To unify the behaviour across multiple parsers, an object defining common test cases has been added.

Reply:
> If this policy is acceptable, other test cases will be changed.

ask:
`userTable` will be defined only here(`src/parser/__tests__/testcase.ts`) ?

Reply:
@hoshinotsuyoshi 

> userTable will be defined only here(src/parser/__tests__/testcase.ts) ?

Yes, I think it's better that way.

Reply:
@MH4GF 
Looks nice 👍
If this policy is accepted, we have to unify type name as well, right?
I would like to know how to change type name for Oracle, SQL Server and so on.

Reply:
@sasamuku Yes, there will inevitably be subtle differences.
I will consider how to override this as we move forward with implementation. 👍🏻